packages/server/src/admin/invite.ts

```diff
@@ -76,8 +76,12 @@ export interface InviteRequest {
 export async function inviteUser(
   request: InviteRequest
 ): Promise<{ user: User; profile: ProfileResource; membership: ProjectMembership }> {
+  if (request.email) {
+    request.email = request.email.toLowerCase();
+  }
+
```

Comment:
Could this logic be added to [createUser](https://github.com/search?q=repo%3Amedplum%2Fmedplum%20createUser&type=code) helper functions?

Reply:
Nice catch -- let me see if that covers everything.